lib/rules/no-empty-component-block.js

```diff
@@ -70,6 +70,9 @@ module.exports = {
 
     return {
       Program() {
+        /** @type {VElement[]} */
+        const emptyBlocks = []
```

Comment:
I think this array can be removed again; better report the errors directly when iterating `componentBlocks`.

Reply:
No worries :slightly_smiling_face: